test/verify/machineslib.py

```diff
@@ -1778,9 +1778,12 @@ def tryCreate(self, dialog):
                 b.wait_present("#vm-{0}-memory".format(name))
                 b.wait_present("#vm-{0}-vcpus".format(name))
 
-            # check memory
-            b.click("#vm-{0}-usage".format(name))
-            b.wait_in_text("tbody.open .listing-ct-body td:nth-child(1) .usage-donut-caption", dialog.getMemoryText())
+            # if OS was specified and memory is 0, virt-install replaces memory value with a value from osinfo
+            # see: virt-manager repo - 5eb47bc00fd75579f94256c272280b26ba0e43f5
+            if dialog.os_vendor != TestMachines.TestCreateConfig.UNSPECIFIED_VENDOR and dialog.memory_size != 0:
```

Comment:
I see your only problem with current behavior is that if memory is rounded down to zero, then virt-install will replace it with it's own value. But that's not functionality we support right?
So le's forbid user to input 0 as memory value. Since it doesn't make sense anyway and virt-manager too doesn't allow user to input 0.
Then we can guarantee that both VM installation will succeed and VM parameters will be what user inputed.
WDYT?

Reply:
Sounds good. You can do it on a follow up PR if you prefer.

Reply:
Sure, I will create another PR which will deal with memory helpblock problems